esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -662,15 +682,15 @@ void EsdlServiceImpl::handleServiceRequest(IEspContext &context,
                                            StringBuffer &logdata,
                                            unsigned int flags)
 {
+    Owned<IEsdlScriptContext> scriptContext;
+
     const char *mthName = mthdef.queryName();
     context.addTraceSummaryValue(LogMin, "method", mthName);
     const char* srvName = srvdef.queryName();
 
     if (m_serviceLevelCrtFail)
         throw MakeStringException(-1, "%s::%s disabled due to Custom Transform errors. Review transform template in configuration.", srvdef.queryName(), mthName);
```

Comment:
You should be able to remove this error checking here. It looks like it is being performed unconditionally inside prepareFinalRequest().

Reply:
will do.

Reply:
(added a comment about why it's also here)